.github/workflows/tests.yml

```diff
@@ -83,6 +83,12 @@ jobs:
           path: ${{ steps.composer-cache.outputs.dir }}
           key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
           restore-keys: ${{ runner.os }}-composer-
+      - name: Install Dusk plugin
```

Comment:
Why don't we pull this in via composer?

Reply:
@LukeTowers we definitely can once it's been added to Packagist. I just wanted to get it working first.